Onboarding/Onboarding/Pages/ProfilePages/Other.cs

```diff
@@ -12,13 +12,33 @@
 
 namespace Onboarding.Pages.ProfilePages
 {
+
     public class Other
     {
-        public string getWecomeText(IWebDriver driver)
+        public IWebDriver driver;
+        Other OtherObj;
+
+        public Other(IWebDriver _driver)
+        {
+            this.driver = _driver;
+            
+        }
+        public void OtherStepDefinitions()
```

Comment:
This I believe is not needed is there any reason you are doing this?

Reply:
redundant